docs/extending.md

```diff
@@ -1,195 +1,36 @@
-# Extending Netlify CMS
-The Netlify CMS exposes an `window.CMS` global object that you can use to register custom widgets, previews and editor plugins. The available methods are:
+## Extending With Widgets
+
+The NetlifyCMS exposes an `window.CMS` global object that you can use to register custom widgets, previews and editor plugins. The available widger extension methods are:
 
-* **registerPreviewStyle** Register a custom stylesheet to use on the preview pane.
-* **registerPreviewTemplate** Registers a template for a collection.
 * **registerWidget** lets you register a custom widget.
 * **registerEditorComponent** lets you add a block component to the Markdown editor
 
-**Writing React Components inline**
```

Comment:
I think we're losing some important content in this file. Any thoughts @bdougie?